scripts/zkai_one_block_executable_package_accounting_gate.py

```
@@ -0,0 +1,658 @@
+    }
+    expected = {
+        "package_without_vk_bytes": EXPECTED_PACKAGE_WITHOUT_VK_BYTES,
+        "package_without_vk_ratio_vs_source": EXPECTED_PACKAGE_WITHOUT_VK_RATIO,
+        "package_without_vk_saving_bytes": EXPECTED_SAVING_WITHOUT_VK_BYTES,
+        "package_with_vk_bytes": EXPECTED_PACKAGE_WITH_VK_BYTES,
+        "package_with_vk_ratio_vs_source": EXPECTED_PACKAGE_WITH_VK_RATIO,
+        "package_with_vk_saving_bytes": EXPECTED_SAVING_WITH_VK_BYTES,
+    }
+    for key, value in expected.items():
+        if summary[key] != value:
+            raise OneBlockPackageAccountingError(f"package summary drift: {key}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)